docs/manage/configuration.md

```diff
@@ -173,6 +184,11 @@ The location where `asdf` will install plugins, shims and tool versions. Can be
 - Default: `$HOME/.asdf`
 - Usage: `export ASDF_DATA_DIR=/home/john_doe/.asdf`
 
+### `ASDF_CONCURRENCY`
+
+- Default: Output of `nproc`, then `sysctl hw.ncpu`, then `/proc/cpuinfo`
```

Comment:
The resultant behaviour of this env var with the `concurrency` config value has these defaults, but this env var defaults to an empty env var, no?

Reply:
Yeah good point - this environment variable defaults to an empty variable. Maybe the code should be modified so that it behaves like other `ASDF_` environment variables and it gets set (internally). Once we do that, we can then also show it on `asdf info`, since I believe it's not printed on there either.